storage_s3/indico_storage_s3/plugin.py

```diff
@@ -96,7 +96,9 @@ def getsize(self, file_id):
 
     def send_file(self, file_id, content_type, filename, inline=True):
         try:
-            fileobj = self.client.get_object(Bucket=self.bucket, Key=file_id)['Body']
-            return send_file(filename, fileobj, content_type, inline=inline)
+            url = self.client.generate_presigned_url('get_object',
+                                                     Params={'Bucket': self.bucket, 'Key': file_id},
+                                                     ExpiresIn=100)
```

Comment:
1m40s is a bit strange ;) I'd use 60 or 120.

Also, we may want to include a content-disposition and content-type header here so we always use the one that's specified when `send_file` is called.